menas/src/main/scala/za/co/absa/enceladus/menas/controllers/OozieController.scala

```diff
@@ -32,6 +34,25 @@ class OozieController @Autowired() (oozieService: OozieService) extends BaseCont
   def isOozieEnabled: Boolean = oozieService.isOozieEnabled
 
   @GetMapping(path = Array("/coordinatorStatus/{id}"))
-  def getCoordinatorStatus(@PathVariable id: String): CompletableFuture[OozieCoordinatorStatus] = oozieService.getCoordinatorStatus(id)  
-  
+  def getCoordinatorStatus(@PathVariable id: String): CompletableFuture[OozieCoordinatorStatus] = {
+    oozieService.getCoordinatorStatus(id)
+  }
+
+  @PostMapping(path = Array("/runNow"))
+  def runNow(@RequestBody schedule: OozieSchedule, @RequestParam reportDate: Optional[String]): CompletableFuture[String] = {
+    val rd: Option[String] = reportDate
```

Comment:
Is this step really needed?

Reply:
Yeah true... it infers the target type from the call